packages/flutter/lib/src/widgets/inherited_model.dart

```diff
@@ -35,13 +35,13 @@ import 'framework.dart';
 /// class MyModel extends InheritedModel<String> {
 ///   const MyModel({super.key, required super.child});
 ///   // ...
-///   static MyModel? of(BuildContext context, String aspect) {
+///   static MyModel? maybeOf(BuildContext context, String aspect) {
```

Comment:
the doc above still mentions the of method. 

Maybe we should actually extend this example and show both, maybeof and of? And also explain the convention here that one is supposed to throw if none can be found?

Reply:
(The docs for InheritedWidget may also be a good place to document the convention that of is supposed to throw if nothing is found and maybeof may return null in that case...)